Engine/Setup/ConsoleSetupHandler.cs

```diff
@@ -168,8 +168,16 @@ public bool Setup(SetupHandlerParameters parameters)
                     isolator.ExecuteWithTimeLimit(TimeSpan.FromMinutes(5),
                         () =>
                         {
-                            //Setup Base Algorithm:
-                            algorithm.Initialize();
+                            try
+                            {
+                                //Setup Base Algorithm:
+                                algorithm.Initialize();
+                            }
+                            catch (Exception err)
+                            {
+                                Log.Error(err);
+                                Errors.Add(new AlgorithmSetupException("During the algorithm initialization, the following exception has occurred: ", err));
```

Comment:
This error was being swallowed by the worker thread, and the backtest wouldn't finish in runtime error